bin/check-rds-events.rb

```diff
@@ -117,7 +117,7 @@ def maint_clusters
         # fetch the last 15 minutes of events for each cluster
         # that way, we're only spammed with persistent notifications that we'd care about.
         clusters.each do |cluster_name|
-          events_record = rds.describe_events(start_time: (Time.now - 900).iso8601, source_type: 'db-instance', source_identifier: cluster_name)
+          events_record = rds.describe_events(start_time: (Time.now - 900).getgm.iso8601, source_type: 'db-instance', source_identifier: cluster_name)
```

Comment:
I was not even aware such a [function](https://ruby-doc.org/core-2.4.0/Time.html#method-i-getgm) existed I generally use [`Time.now.utc`](https://ruby-doc.org/core-2.4.0/Time.html#method-i-utc) which I think is clearer to its intent:
```ruby
irb(main):002:0> Time.now
=> 2018-04-17 18:35:19 -0700
irb(main):003:0> Time.now.utc
=> 2018-04-18 01:35:24 UTC
```